userguide/docs/launchers.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 3
+---
+
+# Launchers and Mod Managers
+
+There are two supported ways that you can launch Minecraft with Neoforge.
+
+The first is to use the official, vanilla launcher from Mojang, which the NeoForge installer creates a profile for.
+
+The second option is to use a third-party launcher, some of which are listed below in the [Third-Party Launchers] section.
+
+## Locating the Game Directory
+
+Minecraft stores its files, like your game's saves, as well as crash reports and log files in a so-called game directory.
+
+The location of the game directory will differ depending on which launcher you use to run the game.
+
+The vanilla launcher will create a `.minecraft` directory in your OS user's home directory.  This is the game directory for the vanilla launcher.
+
+<!-- List ot link out to where to find user home directory? -->
+
+:::note
+If you are using another launcher, refer to their documentation on where the game directory is located.
+:::
+
+## Locating Log Files and Crash Reports
+
+Minecraft stores log files and crash reports in the game directory.
+
+You might be asked to provide log files or crash reports when seeking help.
+
+If you do not know where to find them, open your game directory in a file manager application where there should be two directories:
+
+- `logs` This directory contains log files for the game, with the files `latest.log` and `debug.log` being most pertinent.
+- `crash-reports` This directory contains plain-text crash reports, each with timestamped filenames e.g. `crash-2024-08-11_15.54.06-client.txt`.
+
+<!-- Not sure if the above list is the best way to format the info in it... -->
+
+## Third-Party Launchers
+
+There are a number of third-party launchers (also known as mod managers) to choose from, including:
```

Comment:
I would restructure this subsection as a whole. In my opinion, this should be something along the lines of:

```
There are a number of third-party launchers developed for Minecraft that typically have additional features for modded users. These allow the users to manage different modpacks, install mods themselves etc. /* more info here maybe */

Popular third-party launchers include /* maybe list some pros for each; add links */:

- CurseForge
- Modrinth
- ATLauncher
- Prism Launcher
- FTB?
- GDLauncher
```